packages/mermaid/src/rendering-util/rendering-elements/edges.js

```diff
@@ -556,7 +529,6 @@ export const insertEdge = function (elem, edge, clusterDb, diagramType, startNod
   }
 
   const { x, y } = getLineFunctionsWithOffset(edge);
-  // const lineFunction = edge.curve ? line().x(x).y(y).curve(curve) : roundedCornersLine;
```

Comment:
Leave the lineFunction for now, we can remove it in the  big PR. 
Still working out some issue around it.